apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -386,6 +401,12 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
 
     load_extensions(data)
 
+    allowblank = pl.get_boolean_attrib(
```

Comment:
nit:

```suggestion
    allow_blank = pl.get_boolean_attrib(
```

and related.